innersource-portal.md

```diff
@@ -0,0 +1,48 @@
+
+## Problem
+
+There is no central, easily accessible resource for InnerSource project owners and potential contributors to leverage to advertise or discover InnerSource projects.  
+
+## Story
+
+You are attempting to establish an InnerSource practice within your organization.  You are aware of some projects being run using an InnerSource model, but their existence is only being communicated via word of mouth, email or sidebar conversations with other employees.  There is no single, shared resource for employees across the organization to access which will allow them to easily discover all ongoing InnerSource projects.  This is severely limiting the growth potential of every InnerSource project.  What can be done to help all InnerSource projects raise their visibility to as large an audience as possible and attract contributors organization-wide?
+
+## Forces
+
+* There are multiple engineering teams in your organization but few channels through which they can effectively partner on shared challenges.
+* It is difficult for individuals to discover what InnerSource opportunities exist across the entire company.
+* InnerSource project owners have no clearly defined channel through which to advertise their projects to potential contributors.
+* Individuals have been given some time by their management to get involved in InnerSource projects of their choosing.
+
+## Context
+
+* Managers have given tacit acceptance that their employees should take part in InnerSource projects
+* Employees desire the ability to be made aware of all of the InnerSource project activity occurring across the compnany
+* Owners of InnerSource projects desire the ability to attract as many potential contributors to their projects as possible
+* InnerSource project owners desire full control over the information being communicated concerning their projects
+
+## Solutions
+
+Create an InnerSource Portal intranet website where InnerSource project owners can easily advertise the availability of their projects. Visitors to the InnerSource portal should be able to look across all available projects as well as search for specific projects based on various criteria such as project name, technologies in use, contributor names, sponsoring business unit etc.  The information displayed via the InnerSource portal should be under the full control of the InnerSource project owners at all times and they should have the ability to add or remove projects as needed.  Project owners should include all relevant information concerning their projects within the listings including the owner’s name, the project name and description as well as links to the code repository and any supporting documentation.
```

Comment:
Is information manually entered into the portal? Or is there a way to glean it from a well-formatted README.md file so that it is only maintained in one place and doesn't have to be manually kept up to date?

Reply:
The preferred method would be to source the information displayed via the portal from a data file stored in each project's source code repository. That way the locus of control always remains within the purview of the project owners. We omitted those specifics here as it might have been seen as too prescriptive and implementation specific.  That is definitely up for debate.

Reply:
I think it is helpful to mention it as one option (with the rationale behind it).

Reply:
I was wondering if you also notify projects which have not yet added a meta-file to their repo and help them improve their visibility by adding one. That's something we're currently planning on implementing.

Reply:
Yes. Absolutely.  We are leveraging the internal InnerSource Forum we've created over the past year and asking all of the project owners attending our monthly meetings to add these documents to their repos.